src/components/CasdoorCard/index.js

```diff
@@ -1,20 +1,74 @@
-import React from "react";
-import {useEffect, useState} from "react";
+import Translate from "@docusaurus/Translate";
+import Link from "@docusaurus/Link";
+import React, {useEffect, useState} from "react";
+import axios from "axios";
 
 export default function CasdoorCard(props) {
+  const [loading, setLoading] = useState(false);
   const [link, setLink] = useState("https://door.casdoor.org/");
+  const [mouseState, setMouseState] = useState({state: false});
+  const maskStyle = {
+    position: "absolute",
+    top: "0px",
+    left: "0px",
+    zIndex: "1",
+    height: "100%",
+    width: "100%",
+    background: "rgba(0,0,0,0.4)",
+    cursor: "pointer",
+    borderRadius: "20px",
+  };
   useEffect(() => {
-    setLink(localStorage.getItem("CasdoorLink"));
+    axios.get("https://oa.casbin.com/api/is-mainland-ip").then((response) => {
+      if (response.data === true) {
+        setLink("https://door.casdoor.com/");
+      } else {
+        setLink("https://door.casdoor.org/");
```

Comment:
setLink like this:
![image](https://user-images.githubusercontent.com/53544726/190268012-7f164b92-7dda-48fa-9603-a9fd0cff783a.png)
will help reduce the number of requests